### PR TITLE
Fix compilation errors

### PR DESCRIPTION
**Pull request by josephgbr**
See #10

### CODE REVIEW

File: Makk/M4wII/M4.cpp [hunk 1]
```diff
@@ -2258,13 +2258,15 @@ void CTrack::NoteOn()
 			LevelShift3 = 0;
 
 			while(Frequency*0.5*((float)(2048>>LevelShift3)/pmi->pMasterInfo->SamplesPerSec) > 1.0)
+			{
```
Comment (ensonic) [inline Makk/M4wII/M4.cpp]: please move to prev line

File: Makk/M4wII/M4.cpp [hunk 2]
```diff
@@ -2363,6 +2363,9 @@ inline float CTrack::Osc()
         int B1, B2;
 		int index=0, index2, index3, index4;
 
+        o = 0;
+        o2 = 0;
+
```
Comment (ensonic) [inline Makk/M4wII/M4.cpp]: please add the init to the declaration:
int o = 0, o2 = 0, interp1, ...

**Comment by josephgbr** [comment]
Done.

**Comment by ensonic** [comment]
Thanks!

**Pull request merged**